2021-offseason/src/main/java/frc/robot/RobotContainer.java

```diff
@@ -22,8 +27,13 @@
 
   private final ExampleCommand m_autoCommand = new ExampleCommand(m_exampleSubsystem);
 
+  private final XboxController controller = new XboxController(Constants.kOI.CONTROLLER_PORT);
+
+  private final Intake s_intake;
```

Comment:
maybe a nit ? but section headings would be nice
e.g. // define subsystems or something

Reply:
sounds good